backend/src/components/utils.js

```diff
@@ -251,6 +251,27 @@ function errorResponse(res, msg, code) {
   })
 }
 
+async function postBatches(payload) {
```

Comment:
Can we clean up logging here?

Reply:
The logging in postBatches is consistent with many other similar methods in utils following the same logging pattern... the logging appears helpful torwards log and debugging. Please confirm if you still want removed.

Reply:
Leave for now.